MITADS/qallme_importer.py

```diff
@@ -0,0 +1,80 @@
+import os
+import xml.etree.ElementTree as ET
+from utils import sanitize, line_rules, download
+
+# managing sanitizer
+sanitizer = sanitize.Sanitization()
+
+mapping_normalization = [
+    [u'*', u''],
+    [u'/', u''],
+    [u'#', u''],
+    [u'{', u''],
+    [u'}', u''],
+    [u'(', u''],
+    [u')', u''],
+    [u'[', u''],
+    [u']', u'']
+]
+
+# managing parse directory name
+parsedir = "parsing" + os.path.sep
+
+# managing output pathname + output filename
+outdir = "output" + os.path.sep
+filename = "qallme.txt"
+
+output_file = open(outdir + filename, "w", encoding='utf-8')
+
+# printing a message
+print("\n")
+print("Qallme Importer")
+print("===============")
+# downloading resource
+download_link = 'http://qallme.fbk.eu/archive/QB_IT_V1.0_TranscriptionsReferences.zip'
+
+downloader = download.Download()
+downloader = downloader.if_not_exist(download_link)
+
+# extracting files
+downloader.zip_decompress(parsedir)
```

Comment:
https://github.com/MozillaItalia/DeepSpeech-Italian-Model/blob/master/MITADS/utils/download.py#L59 it is missing the folder where to extract

Reply:
Traceback (most recent call last):
  File "qallme_importer.py", line 43, in <module>
    os.chdir(parsedir + "QB_IT_V1.0_Translations")  # name of the folder inside
FileNotFoundError: [Errno 2] No such file or directory: 'parsing/QB_IT_V1.0_Translations'

Reply:
There is a "kinda-of" bug in download.py
That file has been updated.